languages/elixir/config.json

```
@@ -21,6 +21,12 @@
         "concepts": ["booleans"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "numbers",
+        "uuid": "fee79e03-1496-476f-964f-e60632cb13dc",
+        "concepts": ["floating-point-numbers"],
```

Doesn't the exercise also teach about integers? I know integers are also used in the `basics` exercise, but it feels like they are properly explained in this exercise, so it might be worth adding them as a concept:

```suggestion
        "concepts": ["integers", "floating-point-numbers"],
```